### PR TITLE
Add dsp 1.0.0 manifests.

### PR DESCRIPTION
**Pull request by HumairAK**
Depends on: 
* https://github.com/opendatahub-io/data-science-pipelines-operator/pull/210
* https://github.com/opendatahub-io/data-science-pipelines/pull/110

Related: 
* https://github.com/red-hat-data-services/odh-deployer/pull/364

Live builder image: quay.io/hukhan/rhods-operator-live-catalog:1.31.0-1 

Note I had to use dashboard image: `registry-proxy.engineering.redhat.com/rh-osbs/managed-open-data-hub-odh-operator-bundle-rhel8:v1.31.0-1` because live builder image doesn't seem to work on my device. 

Upstream sources used: 

<details> 

<summary> upstream_sources.yaml </summary> 

```yaml
git:
  # ODH Deployer
  - name: odh-deployer-container-live
    url: https://github.com/DharmitD/odh-deployer
    branch: dspo-labels
    patch: rhods-operator-live/dockerfiles/odh-deployer/Dockerfile

  # ODH Operator Base
  - name: odh-operator-base-container-live
    url: https://github.com/red-hat-data-services/opendatahub-operator
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-operator-base/Dockerfile

  # ODH Operator
  - name: odh-operator-container-live
    url: https://github.com/HumairAK/odh-manifests
    branch: dsp_1-0-0
    patch: rhods-operator-live/dockerfiles/odh-operator/Dockerfile

  # ODH Dashboard
  - name: odh-dashboard-container-live
    url: https://github.com/red-hat-data-services/odh-dashboard
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-dashboard/Dockerfile

  # Kubeflow Notebook Controller
  - name: odh-kf-notebook-controller-container-live
    url: https://github.com/red-hat-data-services/kubeflow
    branch: master
    context: components
    containerfile: notebook-controller/Dockerfile
    patch: rhods-operator-live/dockerfiles/odh-kf-notebook-controller/Dockerfile

  # ODH Notebook Controller
  - name: odh-notebook-controller-container-live
    url: https://github.com/red-hat-data-services/kubeflow
    branch: master
    context: components
    containerfile: odh-notebook-controller/Dockerfile
    patch: rhods-operator-live/dockerfiles/odh-notebook-controller/Dockerfile

  # ODH ML Pipelines Persistence Agent
  - name: odh-ml-pipelines-persistenceagent-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    containerfile: backend/src/Dockerfile.persistenceagent
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-persistenceagent/Dockerfile

  # ODH ML Pipelines API Server
  - name: odh-ml-pipelines-api-server-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    containerfile: backend/src/Dockerfile
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-api-server/Dockerfile

  # ML Pipelines Cacheserver
  - name: odh-ml-pipelines-cache-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-cache/Dockerfile

  # ML Pipelines Scheduled Workflow Server
  - name: odh-ml-pipelines-scheduledworkflow-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-scheduledworkflow/Dockerfile

  # ML Pipelines Artifact Manager
  - name: odh-ml-pipelines-artifact-manager-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-artifact-manager/Dockerfile

  # Data Science Pipelines Operator
  - name: odh-data-science-pipelines-operator-controller-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines-operator
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-data-science-pipelines-operator-controller/Dockerfile

  # ODH Modelmesh Serving Controller
  - name: odh-modelmesh-serving-controller-container-live
    url: https://github.com/red-hat-data-services/modelmesh-serving
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-modelmesh-serving-controller/Dockerfile

  # ODH Model Controller
  - name: odh-model-controller-container-live
    url: https://github.com/red-hat-data-services/odh-model-controller
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-model-controller/Dockerfile

  # ODH Modelmesh Rest Proxy
  - name: odh-mm-rest-proxy-container-live
    url: https://github.com/red-hat-data-services/rest-proxy
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-mm-rest-proxy/Dockerfile

  # ODH Modelmesh Serving Runtime sidecar
  - name: odh-modelmesh-container-live
    url: https://github.com/red-hat-data-services/modelmesh.git
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-modelmesh/Dockerfile

  # ModelMesh Runtime Adapter sidecar
  - name: odh-modelmesh-runtime-adapter-container-live
    url: https://github.com/red-hat-data-services/modelmesh-runtime-adapter
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-modelmesh-runtime-adapter/Dockerfile
    
  # TrustyAI Service
  - name: odh-trustyai-service-container-live
    url: https://github.com/red-hat-data-services/trustyai-explainability
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-trustyai-service/Dockerfile

  # TrustyAI Service
  - name: odh-trustyai-service-operator-container-live
    url: https://github.com/red-hat-data-services/trustyai-service-operator
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-trustyai-service-operator/Dockerfile

```

</details>

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [ ] JIRA link(s):
- [ ] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

File: data-science-pipelines-operator/base/params.env [hunk 1]
```diff
@@ -7,3 +7,6 @@ IMAGES_MOVERESULTSIMAGE=registry.redhat.io/ubi8/ubi-micro:8.7
 IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
 IMAGES_DSPO=registry.redhat.io/rhods/odh-data-science-pipelines-operator-controller-rhel8:latest
 IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
+IMAGES_MLMDENVOY=quay.io/opendatahub/ds-pipelines-metadata-envoy:v1.0.0
+IMAGES_MLMDGRPC=quay.io/opendatahub/ds-pipelines-metadata-grpc:v1.0.0
+IMAGES_MLMDWRITER=quay.io/opendatahub/ds-pipelines-metadata-writer:v1.0.0
```
Comment (HumairAK) [inline data-science-pipelines-operator/base/params.env:12]: Please note that this adding a feature that is not enabled by default as it's incubating upstream, but we need to add the default image values here. These will not be leveraged in rhods.

**Review by DharmitD — COMMENTED** [review]
/lgtm

File: data-science-pipelines-operator/base/params.env [hunk 2]
```diff
@@ -7,3 +7,6 @@ IMAGES_MOVERESULTSIMAGE=registry.redhat.io/ubi8/ubi-micro:8.7
 IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
 IMAGES_DSPO=registry.redhat.io/rhods/odh-data-science-pipelines-operator-controller-rhel8:latest
 IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
+IMAGES_MLMDENVOY=upstream-only
+IMAGES_MLMDGRPC=upstream-only
+IMAGES_MLMDWRITER=upstream-only
```
Comment (HumairAK) [inline data-science-pipelines-operator/base/params.env:12]: This is a downstream specific change, to confirm this doesn't break here's an example of it running using the live builder image provided: 


pr-413

**Review by gmfrasca — APPROVED** [review]

**Pull request merged**